databases/week2/DB_week2_part4/library_creation_script.sql

```diff
@@ -0,0 +1,83 @@
+CREATE SCHEMA IF NOT EXISTS `library` DEFAULT CHARACTER SET utf8 ;
+USE `library` ;
+
+-- category`
+CREATE TABLE IF NOT EXISTS `library`.`category` (
+  `category_id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  `description` VARCHAR(45) NOT NULL,
+  `shelf_nr` INT NOT NULL,
+  PRIMARY KEY (`category_id`))
+ENGINE = InnoDB;
```

Comment:
Remember to be consistent when writing the sql statements

```suggestion
ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```